docs/proposals/organizations.md

```diff
@@ -0,0 +1,118 @@
+
+### Open Questions
+ 1.  Should we have a two layer scoping "/api/v1/organization/<org name>/namespace/<namespace name>" or keep the current namespace structure?
+
+     Keeping the current namespace structure mitigates changes, but prevents two orgs from having the same namespace and makes storing additional org-scoped resources like quota and groups more difficult.
+     I'm in favor of keeping the current structure, but allowing deeper nesting of subresources and having named subresources.
+
+ 2.  Should we allow projects to be owned by Users or only allow ownership by Organizations?
+
+     Allowing Users to own projects eliminates the creation of another "linked" resource that needs to be protected and maintained.
+     Having a single entity allowed to own projects limits some duplication of things like quota.
+
+ 3.  Can we punt on adding users as organization members?
+
+     Organization-admins should not be able to force users into an Org.  Users should not be able to join any Org they want.  I'd like to get the structure correct and punt on the ability to add members.  It would require some sort of invitation/request/approval flow.
+
+ 4.  Can we punt on project transfering?
+
+     An org-admin shouldn't be able to take a project from someone else and they shouldn't be allowed to orphan their own project.  That means we'd need an offer/request/approval flow.
+
+
+
+## Organizations
+An Organization is a cluster scoped API object that references Users and has subresources for accessing and modifying Organization scoped OrgGroups.
+```
+// Organization contains the set of Users that are members of the Organization.
+type Organization struct {
+	kapi.TypeMeta
+	kapi.ObjectMeta
+
+	// AdminUsers is a list of Users that can administer this Organization.  By default a policy rule exists
+	// that allows all Users to inspect and modify OrgGroups of Organizations that they are an Admin for.
+	// Admins may NOT modify Organization membership.
+	AdminUsers util.StringSet
+	// AdminGroups is a list of Groups that can administer this Organization.  By default a policy rule exists
+	// that allows all Users to inspect and modify OrgGroups of Organizations that they are an Admin for.
+	// Admins may NOT modify Organization membership.
+	AdminGroups util.StringSet
+
+	// MemberUsers is a list of Users that are members of the Organization.  Only members may be assigned to OrgGroups.
+	MemberUsers util.StringSet
+	// MemberGroups is a list of Groups whose constituent users are considered members of the Organization.  Only members may be assigned to OrgGroups.
+	MemberGroups util.StringSet
+}
+```
+
+The Organization has a list of admins.  Admins have the power to:
+ 1.  Reserve quota for particular projects.
+ 2.  Accept ownership of a project.
+ 3.  Offer ownership of a project.
+ 4.  Remove (but not add) group members.
```

Comment:
Maybe feels like named subresources, which we don't have... with two seconds thought, something like:
```
create org/myorg/members {name:bob}
delete org/myorg/members/bob
create user/~/orgs {name:myorg}
delete user/~/orgs/myorg
```